user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -500,36 +536,53 @@ define([
 
     updateData: function () {
       var myself = this;
+
+      const trashFolder = {
+        "file": {
+          "trash": "trash",
+          "createdDate": "1365427106132",
+          "fileSize": "0",
+          "folder": true,
+          "hidden": "false",
+          "objectId:": jQuery.i18n.prop('trash'),
+          "locale": "en",
+          "locked": "false",
+          "name": jQuery.i18n.prop('trash'),
+          "ownerType": "-1",
+          "path": ".trash",
+          "title": jQuery.i18n.prop('trash'),
+          "versioned": "false"
+        }
+      };
+
       myself.set("runSpinner", true);
-      myself.fetchData("/", function (response) {
-        var trash = {
-          "file": {
-            "trash": "trash",
-            "createdDate": "1365427106132",
-            "fileSize": "0",
-            "folder": "true",
-            "hidden": "false",
-            "id:": jQuery.i18n.prop('trash'),
-            "locale": "en",
-            "locked": "false",
-            "name": jQuery.i18n.prop('trash'),
-            "ownerType": "-1",
-            "path": ".trash",
-            "title": jQuery.i18n.prop('trash'),
-            "versioned": "false"
+      FileBrowser.clearTreeCache();
```

Comment:
Consider moving the `FileBrowser.clearTreeCache();` calls to inside `fetchTreeRootData`.